js/apps/admin-ui/cypress/e2e/user_fed_ldap_test.spec.ts

```diff
@@ -97,11 +100,22 @@ const ldapTestFailMsg =
   "Error when trying to connect to LDAP: 'CommunicationError'";
 
 describe("User Federation LDAP tests", () => {
+  const realmName = `ldap-realm-${uuid()}`;
+
+  before(() => {
+    adminClient.createRealm(realmName);
+  });
```

Comment:
This will not work, as it doesn't actually return the Promise. If an arrow function has a body with brackets then it will not implicitly return the value, so a `return` statement must be added, or the brackets removed.
```suggestion
  before(() => adminClient.createRealm(realmName));
```